tests/js/testUtils.js

```diff
@@ -28,4 +28,66 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             });
         });
     };
+
+    // A component for running tests that need variables loaded
+    // from an external JSON file, but that don't otherwise
+    // need asynchronous testing
+    fluid.defaults("floe.tests.utils.JSONFixtureTester", {
```

Comment:
The infusion built-in component ```fluid.prefs.resourceLoader``` already does this. You've already used it for template loading @ https://github.com/waharnum/chartAuthoring/blob/GPII-1584/src/js/chartAuthoring.js#L28 for loading templates. It can also be used for loading json contents.